lib/ansible/plugins/strategy/__init__.py

```diff
@@ -484,20 +484,25 @@ def _process_pending_results(self, iterator, one_pass=False, max_passes=None, do
         '''
 
         ret_results = []
-        handler_templar = Templar(self._loader)
+        self._handler_templar = None
+
+        def handler_templar():
```

Comment:
if you make it into a getter you can even leave most of the code as is

Reply:
Thanks for the comment.

I had a couple of versions for the fix, this one uses `@property`: https://github.com/ansible/ansible/commit/5ae7c3596731c5a3e32235dbf43de7543e04b86f

In the end I went for the solution which is restricted to the method `_process_pending_results`. I don't really have a preference, though.